.github/workflows/check-wordpress-versions.yml

```diff
@@ -127,3 +127,13 @@ jobs:
         if: env.CHANGES_DETECTED == 'true'
         run: |
           git push origin HEAD:refs/heads/update-plugins-and-wordpress
+
+      # Create a pull request if there are changes
+      - name: Create Pull Request
+        if: env.CHANGES_DETECTED == 'true'
+        uses: peter-evans/create-pull-request@v6
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          branch: update-plugins-and-wordpress
+          title: 'Update WordPress Docker and WooCommerce Versions'
```

Comment:
Afaik, it is not only WooCommerce, but it could also be other plugins, right?

Reply:
This title is a bit tricky because it can be all plugins without WP or only WC, etc.
Maybe Porter can update it. We could generate it, but I am not sure if it is worth it.

Reply:
It is probly not worth it. I would keep it as is. In case we find it confusing we can use something more generic like "update versions in Circle CI configuration"